### ISSUE TITLE
Picking up an Egg from 2 Pokes I own causes random OW Pokemon to get mad

### ISSUE THREAD

**Issue by PigmyRhino** [SheerSt/pokewilds#281]
I added a Krabby and a Corsola to my party, gave them a nice beach habitat and they made an egg.

When I collected the egg, an OW Totodile, OW Wartortle, and OW Staryu got pissed at me.. Should that happen?

**Comment by SiriusGG** [comment]
Were the Totodile, Wartortle and Staryu in the approximate area where you got Krabby or Corsola or in a completely different location? Is it possible, that Krabby and Corsola previously made eggs with them? Because if so it probably got something to do with it.

**Comment by PigmyRhino** [comment]
Yes, the immediate area. I didn't think of them making eggs with the other OW Pokemon. Thanks! That's probably what happened.
